backend/src/auth_service/privacy_pass.rs

```diff
@@ -27,6 +29,7 @@ impl<'a, 'b> AuthServiceBatchedKeyStoreProvider<'a, 'b> {
 #[async_trait]
 impl BatchedKeyStore for AuthServiceBatchedKeyStoreProvider<'_, '_> {
     /// Inserts a keypair with a given `truncated_token_key_id` into the key store.
+    // TODO: What is the semantics on collision?
```

Comment:
I'd say if we try to store the same key twice, something has gone wrong and it should log an error. We don't have control over the trait here (although we could ask @raphaelrobert if we wants to change the interface over in the `privacypass` crate), so I think logging the error is all we can do.

Reply:
Added a test to check that double insert keeps the value from the first and throws away the second value.